src/Og.php

```diff
@@ -273,4 +288,36 @@ protected static function fieldInfo($field_name = NULL) {
     return $fields_config;
   }
 
+  /**
+   * Get the selection handler for an audience field attached to entity.
+   *
+   * @param $entity
+   *   The entity type.
+   * @param $bundle
+   *   The bundle name.
+   * @param $field_name
+   *   The field name.
+   * @param array $options
+   *   Overriding the default options of the selection handler.
+   *
+   * @return OgSelection
+   * @throws \Exception
+   */
+  public static function getSelectionHandler($entity, $bundle, $field_name, array $options = []) {
```

Comment:
I don't think we need an $option, just `$field_mode` or even `$is_admin = FALSE`, as In the OG complex widget we'll get the right selection handler without mocking the `handler_settings`